LICENSE

```diff
@@ -0,0 +1,22 @@
+Copyright 2014-2016 Eric Davis and Homebrew contributors.
```

Comment:
@edavis For simplicity would you mind just being included under `Homebrew contributors` here?